loops-arrays-objects.js

```diff
@@ -32,9 +32,24 @@ const scores = [
 // D
 // ==========================================
 
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    let score = scores[i].score
+    if (score < 60) {
+        console.log('F')
+    }
+    else if (score >= 60 && score < 70) {
+        console.log('D')
+    }
+    else if (score >= 70 && score < 80) {
+        console.log('C')
+    }
+    else if (score >= 80 && score < 90) {
+        console.log('B')
+    }
+    else if (score >= 90 && score < 100) {
```

Comment:
deze zou je dan ook als else kunnen zetten, omdat dan alles wat 90 en hoger is nog overblijft.